### DISCUSSION TITLE
Hydration Middleware Error - undefined index "id" on "payload"

### DISCUSSION THREAD

**Discussion by bzarzuela** [livewire/livewire#4851] [7. Bugs (archived)]
In line 27 of vendor/livewire/livewire/src/HydrationMiddleware/PerformActionCalls.php, there is an assignment that fails when a component is being hydrated. 

<img width="626" alt="image" src="https://user-images.githubusercontent.com/230894/160956029-883769e4-0f13-46c1-8ede-4ce13c299ebd.png">

I don't know exactly what causes this, it only happens in our production app that's using Flare to capture errors. My theory is that it could be a really stale browser tab that somehow refreshed itself. I don't really know. 

In our apps, we have to do this in Exceptions\Handler:

```php
$this->reportable(function (ErrorException $e) {
    if ($e->getMessage() === 'Undefined array key "id"') {
        return false; // Temporary fix for the Livewire Hydration problem
    }
});
```

If there's something we're doing wrong, please let me know. Otherwise, I'll open PR to check if that key is set before proceeding with the code.

**Comment by joshhanley** [comment]
@bzarzuela please see this pinned discussion for more details https://github.com/livewire/livewire/discussions/3769

**Reply by bzarzuela** [reply]
Thanks for the suggestion, @joshhanley. Unfortunately, it doesn't apply to me because I didn't publish the Livewire Javascript assets in the first place.

My theory is that it could be a browser that still has outdated assets loaded. Like a tab that was never closed but gets refreshed in the background. Do you think this possible?
